modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/snapshot/storage/SnapshotLocalFileSystemStorage.scala

```diff
@@ -50,6 +51,9 @@ final class SnapshotLocalFileSystemStorage[F[_]: Async: KryoSerializer, S <: Sna
   def read(hash: Hash): F[Option[Signed[S]]] =
     read(hash.coerce[String])
 
+  def readHash(ordinal: SnapshotOrdinal): F[Option[Hash]] =
+    read(ordinal).flatMap(_.map(_.toHashed.map(_.hash)).traverse(identity))
```

Comment:
as above, traverse with identity is just a sequence

Reply:
Changed

```scala
read(ordinal).flatMap(_.map(_.toHashed.map(_.hash)).traverse(identity))
```

to

```scala
read(ordinal).flatMap(_.map(_.toHashed.map(_.hash)).sequence)
```

Reply:
and as written in many other comments - map + sequence = traverse

Reply:
This has been removed since it is not being used by an endpoint.